jekyll/_cci2/migrating-from-launch-agent-to-the-new-machine-runner-agent.adoc

```diff
@@ -0,0 +1,165 @@
+---
+contentTags: 
+  platform:
+  - Cloud
```

Comment:
Is this definitely not applicable to server?

Reply:
Yes, Machine Runner 3.0 will not be included in the upcoming server release. We will update the docs to also include server once Machine Runner 3.0 is supported on server.